matrix-meetings-widget/src/components/meetings/MemberSelectionDropdown/MemberSelectionDropdown.tsx

```diff
@@ -244,53 +228,28 @@ export function MemberSelectionDropdown({
             ...props.InputProps,
             margin: 'dense',
             'aria-describedby': instructionId,
-            endAdornment: (
-              <>
-                {showSelectAll && (
-                  <InputAdornment
-                    position="end"
-                    sx={{
-                      right: 60,
-                      top: 'calc(50%)',
-                      position: 'absolute',
-                    }}
-                  >
-                    <Button
-                      color="inherit"
-                      disabled={canSelectAll}
-                      onClick={handleSelectAll}
-                    >
-                      {t('memberSelectionDropdown.selectAll', 'Select all')}
-                    </Button>
-                  </InputAdornment>
-                )}
-                {props.InputProps.endAdornment}
-              </>
-            ),
           }}
           label={label}
           size="medium"
         />
       );
     },
-    [canSelectAll, handleSelectAll, instructionId, label, showSelectAll, t]
+    [instructionId, label]
   );
 
   const getOptionLabel = useCallback(
-    (o: StateEvent<RoomMemberStateEventContent>) =>
-      o.content.displayname ?? o.state_key,
+    (o: MemberSelection) => o.content.displayname ?? o.state_key,
```

Comment:
We can think about making `displayName` a required field in the interface so we don't need to have this logic in this component.

```suggestion
    (o: MemberSelection) => o.displayName,
```

This also applies to a lot of `... ?? undefined` calls for displayname and avatar_url.

But this might also make the usage of this component more complex, so maybe it's a bad idea. Do what you would prefer 🙂.

Reply:
After normalization there are no `?? undefined` calls inside of `MemberSelectionDropdown` so  it looks better now, `displayName` is not required still but I it is okay from my point of view.